docs/product/base/grid.html

```diff
@@ -0,0 +1,816 @@
+---
+layout: page
+title: Grid
+description: Atomic CSS grid classes allow you to quickly add native css grids to your container.
+---
+<section class="stacks-section">
+    {% header "h2", "Overview" %}
+    {% tip, "warning", "mb24" %}
+        Looking for our old flex grid? <a href="{{ "/product/base/flex" | relative_url }}">Flex layouts</a> have been moved and renamed now that CSS grid is here. Our <code class="stacks-code">.grid</code>, and <code class="stacks-code">.grid--cell</code> classes have been removed in favor of <code class="stacks-code">.d-flex</code> and <code class="stacks-code">.flex--item</code>.
+    {% endtip %}
+
+    <p class="stacks-copy">CSS Grids are the most powerful layout system available in CSS. It has two dimensions, meaning it can handle both columns and rows simultaneously, unlike <a href="{{ "/product/base/flex" | relative_url }}">flex layouts</a> which can only do one at a time. Applying <code class="stacks-code">.d-grid</code> to a container will lay out its children according to the CSS Grid layout spec. Adding atomic modifying classes will change the layout’s behavior. Applying classes to an individual <code class="stacks-code">.grid--item</code> will change that cell’s behavior.</p>
+
+    {% header "h3", "Examples" %}
+    <div class="stacks-preview">
+{% highlight html %}
+<div class="d-grid">
+    <div class="grid--item">…</div>
+</div>
+{% endhighlight %}
+        <div class="stacks-preview--example ff-mono">
+            <div>.d-grid</div>
+            <div>.grid__4</div>
+            <div>.g16</div>
+            <div class="d-none sm:d-inline-block sm:highlight">.sm:g-af-row</div>
```

Comment:
This portion of the docs read as a mistake to me at first glance. It took me a minute to realize that these classes are applied to the layout.

I'd suggest giving the entire example some border/background-color/padding (or just ignore if you've got bigger 🦐 to fry).

Reply:
Valid feedbak, and I think I've felt the same revisiting it after all this time.

Reply:
huh, I had a code suggestion, but it got eaten by Github.
 
In any case:
```suggestion
            <div class="mb16">
                .d-grid .grid__4 .g16
                <div class="d-none sm:d-inline-block sm:highlight">.sm:g-af-row</div>
            </div>
```